### PR TITLE
bootstrap database from gradle

### PR DESCRIPTION
**Pull request by CyberDem0n**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/83" target="_blank">ReviewNinja

File: build.gradle [hunk 1]
```diff
@@ -150,6 +152,62 @@ task wrapper(type: Wrapper) {
...
+}
+
+def getSqlInstance() {
+    // This is needed to get postgreSQL driver onto the Groovy/Gradle classpath
+    configurations.pgsql.each { file ->
+        println "Adding URL: $file"
+        gradle.class.classLoader.addURL(file.toURI().toURL())
+    }
+
+    def db = [
```
Comment (v-stepanov) [inline build.gradle]: just wondering if we can somehow grab those settings from https://github.com/zalando/nakadi/blob/ARUHA-50-db-bootstrap/src/main/resources/application.yml#L8 to have that in one place only

Comment (jkliff) [inline build.gradle]: +1000
We certainly can and doing so would be quite handy to centralize all the Nakadi "manual tasks" on gradle.

File: database/schemaregistry/10_data/04_tables/01_event_type.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS zsr_data.event_type (
+  et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_event_type_object jsonb NOT NULL
```
Comment (jkliff) [inline database/schemaregistry/10_data/04_tables/01_event_type.sql]: probably would not be bad having a further constraint checking that the name field (within the event type object) matches et_name.

File: src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java [hunk 3]
```diff
@@ -44,11 +44,7 @@ public void setUp() {
             repository = new EventTypeDbRepository(template, mapper);
             connection = datasource.getConnection();
 
-            ResourceDatabasePopulator rdp = new ResourceDatabasePopulator();
-            rdp.addScript(new ClassPathResource("schema.sql"));
-            rdp.populate(connection);
-
-            template.execute("TRUNCATE TABLE zsr_data.event_type");
+            template.execute("DELETE FROM zsr_data.event_type");
```
Comment (jkliff) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java]: why before?

File: src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java [hunk 4]
```diff
@@ -44,11 +44,7 @@ public void setUp() {
             repository = new EventTypeDbRepository(template, mapper);
             connection = datasource.getConnection();
 
-            ResourceDatabasePopulator rdp = new ResourceDatabasePopulator();
-            rdp.addScript(new ClassPathResource("schema.sql"));
-            rdp.populate(connection);
-
-            template.execute("TRUNCATE TABLE zsr_data.event_type");
+            template.execute("DELETE FROM zn_data.event_type");
```
Comment (rcillo) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java]: I'm gonna fix this once we merge this PR into the base branch. I need to add `DELETE` to other tests which were affecting this one. We can leave it here by now.

**Pull request merged**